### ISSUE TITLE
Enable dependabot for this project

### ISSUE THREAD

**Issue by Turbo87** [lerna/lerna-changelog#148]
I've been using https://dependabot.com/ on several other projects and found it quite useful. I would like to enable it for this project too, but it looks like only org admins can do so.

@hzoo any thoughts on this?

**Comment by trivikr** [comment]
I've also been using dependabot on my other projects and have found it helpful.
The only issue I see is that lerna-changelog is updated rarely. The current 0.x version was released in Oct 2018, and still supports Node.js 6.x which was EoL in March 2018.

The CIs will fail when dependabot will update dependencies with new major versions which deprecate old Node.js versions, and you'll have ignore those major version updates. Dependabot is a good choice, if you plan to follow Node.js release schedule for your latest major versions (i.e. come up with lerna-changelog new major version when a Node.js LTS gets EoL)

**Comment by Turbo87** [comment]
yeah, I totally agree @trivikr. lerna-changelog for me is just one of many side projects so my time for this is limited. but I admit that I have updated a lot of other projects already where I dropped Node 6 support, mostly because the dependabot PRs were starting to all be red. so by adding dependabot here it would create a bit more pressure to keep up with the Node release schedule 😉

**Comment by trivikr** [comment]
> by adding dependabot here it would create a bit more pressure to keep up with the Node release schedule 😉

🎉

**Comment by hzoo** [comment]
Ok enabled it, not sure if we need to change something else?

**Comment by trivikr** [comment]
> not sure if we need to change something else?

As this project has package.json, @dependabot will start cutting PRs updating dependencies as per the chosen frequency (I prefer weekly updates in my projects)

Since lerna-changelog CI still tests for Node.js 6.x, it'll be red for many dependabot PRs unless we deprecate 6.x as suggested in https://github.com/lerna/lerna-changelog/issues/150
